tachyon/zk/plonk/examples/fibonacci/BUILD.bazel

```diff
@@ -73,7 +73,6 @@ tachyon_cc_library(
 
 tachyon_cc_unittest(
     name = "fibonacci1_circuit_test",
-    testonly = True,
```

Comment:
Why is it redundant?

Reply:
I think it's because it's a `tachyon_cc_unittest` which ensures it's a test lol.

Reply:
If you check the [Bazel documentation](https://bazel.build/reference/be/common-definitions#typical-attributes), you will find the following:

Tests (`*_test` rules) and test suites (`test_suite` rules) are `testonly` by default.

Additional reference:

```python
def tachyon_cc_unittest(
        name,
        size = "small",
        **kwargs):
    tachyon_cc_test(
        name = name,
        size = size,
        **kwargs
    )
    
def tachyon_cc_test(
        name,
        copts = [],
        defines = [],
        local_defines = [],
        linkopts = [],
        linkstatic = True,
        deps = [],
        safe_code = True,
        force_exceptions = False,
        force_rtti = False,
        **kwargs):
    cc_test(
        name = name,
        copts = copts + tachyon_cxxopts(safe_code = safe_code, force_exceptions = force_exceptions, force_rtti = force_rtti),
        defines = defines + tachyon_defines(),
        local_defines = local_defines + tachyon_local_defines(),
        linkopts = linkopts + tachyon_linkopts(),
        linkstatic = linkstatic,
        deps = deps + ["@com_google_googletest//:gtest_main"],
        **kwargs
    )
```